hw/hw3/ex_02.py

```diff
@@ -0,0 +1,68 @@
+# Задача 18: Требуется найти в массиве A[1..N] самый близкий по величине элемент к заданному числу X. Пользователь в первой строке вводит натуральное число N – количество элементов в массиве. В последующих  строках записаны N целых чисел Ai. Последняя строка содержит число X
+
+# *Пример:*
+
+# 5
+#     1 2 3 4 5
+#     6
+#     -> 5
+
+n = int(input("введите колво элтов: -- "))
+a = []
+for i in range(n):
+    a.append(int(input(f"введите {i + 1} элт массива: -- ")))
+
+x = int(input("введи число для поиска самого близкого по величине элемент: -- "))   
+
+
+dict = {} # библиотека ключ разница    value-элт массива
+for i in range(n):
```

Comment:
Можно проходиться напрямую по элементам списка.
```suggestion
for i in a:
```